source/_integrations/netgear.markdown

```diff
@@ -35,7 +35,10 @@ Consider_home:
 {% endconfiguration_basic %}
 
 ## Router entities
-The NETGEAR router will have the following entities:
+The NETGEAR router will have the following entities.
+Note that not all routers support all features, if a router does not support a feature the corresponding entity will have the unavailable status even when the enity is disabled.
+You might also see the following error in the log `404 service '...', method '...' not found`, to prevent these errors, keep the unsuported entities disabled.
+All posibly unsupported enities are disabled by default.
```

Comment:
```suggestion

Note that not all routers support all features, if a router does not support a feature, the corresponding entity will have the unavailable status even when the entity is disabled.

You might also see the following error in the log `404 service '...', method '...' not found`, to prevent these errors, keep the unsupported entities disabled.

All possibly unsupported entities are disabled by default.
```